### DISCUSSION TITLE
Unable to Access 05 and Above on ZTM

### DISCUSSION THREAD

**Discussion by AaronGWang** [mrdbourke/pytorch-deep-learning#851] [Q&A] [answered]
I went on ZTM to access the rest of the course, but am unable to submit payment info and buy the subscription. Is the rest of the course available somewhere else? I tried to join the ZTM discord but it is private to subscribers, and they do not have a help center. 

Does anyone have any ideas?

**Comment by mrdbourke** [comment, marked as answer]
Hey @AaronGWang , 

Sorry to hear about this! 

Have you reached out to support@zerotomastery.io? 

They should be able to help with subscription issues to get you access to sections 05 and above as well as the ZTM Discord.

Otherwise, the ZTM PyTorch course is also available on Udemy with all of the same content: https://www.udemy.com/course/pytorch-for-deep-learning/

**Reply by AaronGWang** [reply]
Thank you for the swift reply, I have now purchased the course through Udemy! 

Just want to thank you for responding so quickly, as well as the course itself. The most comprehensive and hands-on course I am and have ever taken. Helped me get started on much larger-scale projects, so props to you!

**Reply by mrdbourke** [reply]
You're welcome! Enjoy the course and let me know if there's anything I can help with :)
